CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## 7.5.0 - 2024-06-13
```

Comment:
I know this is already merged, but should this date be updated? @tylermarien 😅

Reply:
> I know this is already merged, but should this date be updated? 😅

It's not too critical to be exact there. I just update it when I make the changes and don't worry too much about when it is merged.